packages/components/RadioGroup/src/RadioButton.tsx

```diff
@@ -54,13 +61,16 @@ export const RadioButton = compose<IRadioButtonType>({
       if (!state.selected) {
         info.onChange && info.onChange(buttonKey);
       }
+      setIsFocused(true);
```

Comment:
So basically what I'm saying is: 

`const onFocusChange = React.useCallback((ev: NativeSyntheticEvent<{}> ) => {
      if (!state.selected) {
        info.onChange && info.onChange(buttonKey);
      }
      pressable.props.onFocus && pressable.props.onFocus(ev);
    }, [state, pressable.props, info, buttonKey]);`

Reply:
I believe we tried this when I first developed RadioGroup and it didn't work (I can't remember why exactly). I'm trying it now though, but I'm getting an error here: `pressable.props.onFocus(ev)` stating mismatching parameters. Going to look into it more now. Is the NativeSyntheticEvent the correct event to be passing in?

Reply:
Thank you for the advice, changed in newest commit